packages/tsconfig.json

```diff
@@ -0,0 +1,71 @@
+{
+  "files": [],
+  "include": [],
+  "references": [
+    {"path": "./address"},
+    {"path": "./address-consts"},
+    {"path": "./address-mocks"},
+    {"path": "./admin-graphql-api-utilities"},
+    {"path": "./ast-utilities"},
+    {"path": "./async"},
+    {"path": "./csrf-token-fetcher"},
+    {"path": "./css-utilities"},
+    {"path": "./dates"},
+    {"path": "./decorators"},
+    {"path": "./enzyme-utilities"},
+    {"path": "./function-enhancers"},
+    {"path": "./graphql-persisted"},
+    {"path": "./graphql-testing"},
+    {"path": "./i18n"},
+    {"path": "./jest-dom-mocks"},
+    {"path": "./jest-koa-mocks"},
+    {"path": "./jest-mock-apollo"},
+    {"path": "./jest-mock-router"},
+    {"path": "./koa-liveness-ping"},
+    {"path": "./koa-metrics"},
+    {"path": "./koa-shopify-auth"},
+    {"path": "./koa-shopify-graphql-proxy"},
+    {"path": "./koa-shopify-webhooks"},
+    {"path": "./logger"},
+    {"path": "./network"},
+    {"path": "./performance"},
+    {"path": "./polyfills"},
+    {"path": "./predicates"},
+    {"path": "./react-app-bridge-universal-provider"},
+    {"path": "./react-async"},
+    {"path": "./react-compose"},
+    {"path": "./react-csrf"},
+    {"path": "./react-csrf-universal-provider"},
+    {"path": "./react-effect"},
+    {"path": "./react-effect-apollo"},
+    {"path": "./react-form"},
+    {"path": "./react-form-state"},
+    {"path": "./react-google-analytics"},
+    {"path": "./react-graphql"},
+    {"path": "./react-graphql-universal-provider"},
+    {"path": "./react-hooks"},
+    {"path": "./react-html"},
+    {"path": "./react-hydrate"},
+    {"path": "./react-i18n"},
+    {"path": "./react-i18n-universal-provider"},
+    {"path": "./react-idle"},
+    {"path": "./react-import-remote"},
+    {"path": "./react-intersection-observer"},
+    {"path": "./react-network"},
+    {"path": "./react-preconnect"},
+    {"path": "./react-router"},
+    {"path": "./react-self-serializers"},
+    {"path": "./react-serialize"},
+    {"path": "./react-server"},
+    {"path": "./react-server-webpack-plugin"},
+    {"path": "./react-shopify-app-route-propagator"},
+    {"path": "./react-shortcuts"},
+    {"path": "./react-testing"},
+    {"path": "./react-tracking-pixel"},
+    {"path": "./react-universal-provider"},
+    {"path": "./semaphore"},
+    {"path": "./sewing-kit-koa"},
+    {"path": "./useful-types"},
+    {"path": "./with-env"}
```

Comment:
It sucks that we have to manually maintain a list like this. It doesn't have to be part of this PR but at the least we should add a task to include adding an entry here in the new package generator, or at least have a separate script that gets run after like we do with adding to the main readme.

Reply:
I've added a [test](https://github.com/Shopify/quilt/pull/962/files#diff-09d775e5b319188b96e0907456993c34) to cover for now.